appengine-standard-java8/helloworld-gae-launch-dataflow/README.md

```diff
@@ -61,13 +61,13 @@ detailed instructions.
 ## Maven
 ### Running locally
 
-    mvn appengine:run
+    mvn package appengine:run
```

Comment:
I used to do this all the time, but I found that package happened automatically when I used appengine:run.  That said, I don't see the need to remove it.

Reply:
With the new updates to the plugin, you have to package first.